x-pack/plugins/security_solution/common/endpoint/service/trusted_apps/validations.ts

```diff
@@ -34,6 +34,36 @@ export const getDuplicateFields = (entries: ConditionEntry[]) => {
     .map((entry) => entry[0]);
 };
 
+const WIN_EXEC_PATH = /\\(\w+\.\w+)$/i;
```

Comment:
These regexps doesn't cover many cases that are possible, e.g.:  processes with special characters like symbols, spaces, etc... see https://regex101.com/r/9tH87v/1

Reply:
from our slack discussion, consider using this instead:

POSIX paths (linux, mac)

```typescript
path.basename(path)
```

Windows paths:

```typescript
path.win32.basename(path)
```

Reply:
Actually - I take that back. You can't use `path` here because this can be used on the client side (UI) as well (is a module under `common/`). So you might need to use regexp or look for an already installed library that we can import client side